### PR TITLE
Edit documentation

### PR DESCRIPTION
**Pull request by addempsea**

### CODE REVIEW

File: lib/nibss/README.md [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
+Credential is gotten from nibss. Prior to acquiring this credential, 11111 should be used for testing.
 
 ##### host
+
 This argument is optional in all cases. Defaults to `https://sandboxapi.fsi.ng` if not found.
 
 ### nibss.Bvnr.Reset(credentials)
-Returns the aes_key, ivey and password. As earlier stated this function should be called at every initialization. The credentials returned is set as part of the data for accessing other nibss modules.
+
+Returns the aes_key, ivey and password as part of the response header. As stated earlier this function should be called at every initialization. The credentials returned are set as part of the data for accessing other nibss modules.
```
Comment (akinjide) [inline lib/nibss/README.md]: ivey should be ivkey

File: lib/nibss/README.md [hunk 2]
```diff
@@ -1,42 +1,54 @@
...
 The following example calls the reset module
+
 ```javascript
-nibss.Bvnr.Reset({sandbox_key:'Your sandbox_key', organisation_code: 'Your organisation_code'})
-.then((response) => {
-    //do something with the response here
-})
+nibss.Bvnr.Reset({
+  sandbox_key: "Your sandbox_key",
+  organisation_code: "Your organisation_code" || "11111"
```
Comment (akinjide) [inline lib/nibss/README.md]: "Your organisation_code" replace with "organisation_code from sandbox dashboard" and apply everywhere necessary

File: lib/nibss/README.md [hunk 3]
```diff
@@ -172,96 +198,101 @@ fingerPrintData: {
...
+    MiddleName: "Owner Middle Name" || "Adepoju",
+    AccountNumber: "Owner Account Number" || "0987654321",
+    BankCode: "Bank Code" || "011"
+  },
+  sandbox_key: "Your sandbox_key",
+  organisation_code: "Your organisation_code" || "11111",
+  password: "Your password",
+  ivkey: "Your ivkey",
+  aes_key: "Your aes_key",
+  host: "Your host url" || null
```
Comment (akinjide) [inline lib/nibss/README.md]: Why do we have texts highlighted red

File: test/fixtures/Transfer.js [hunk 4]
```diff
@@ -1,33 +1,39 @@
...
+     NEResponse: faker.random.number(100),
+     BenefiName: faker.random.number(100),
+     PaymentReference: faker.random.number(100),
+     OriginatorAccountName: faker.random.number(100),
+     translocation: faker.random.number(100)
+    },
+    subscription_key: "t",
+    Appid: faker.random.number(100),
+    ipval: 0
+  }
```
Comment (akinjide) [inline test/fixtures/Transfer.js]: Inconsistent indentation spacing

File: lib/nibss/README.md (hunk 1 above)
Comment (addempsea) [reply lib/nibss/README.md]: Changed.

File: lib/nibss/README.md (hunk 2 above)
Comment (addempsea) [reply lib/nibss/README.md]: Applied everywhere.

File: lib/nibss/README.md (hunk 3 above)
Comment (addempsea) [reply lib/nibss/README.md]: Fixed.

File: test/fixtures/Transfer.js (hunk 4 above)
Comment (addempsea) [reply test/fixtures/Transfer.js]: Fixed in the other pr

File: lib/nibss/README.md (hunk 3 above)
Comment (addempsea) [reply lib/nibss/README.md]: Not fixed.

Comment (addempsea) [reply lib/nibss/README.md]: Fixed

**Pull request merged**